api/layer/container.go

```diff
@@ -154,25 +155,12 @@ func (n *layer) setContainerEACL(ctx context.Context, cid *cid.ID, gateKey *keys
 	return nil
 }
 
-func formDefaultTable(cid *cid.ID, gateKey ecdsa.PublicKey) *eacl.Table {
-	table := eacl.NewTable()
-	table.SetCID(cid)
-
-	for op := eacl.OperationGet; op <= eacl.OperationRangeHash; op++ {
-		record := eacl.NewRecord()
-		record.SetOperation(op)
-		record.SetAction(eacl.ActionAllow)
-		eacl.AddFormedTarget(record, eacl.RoleUser, gateKey)
-		table.AddRecord(record)
-
-		record2 := eacl.NewRecord()
-		record2.SetOperation(op)
-		record2.SetAction(eacl.ActionDeny)
-		eacl.AddFormedTarget(record2, eacl.RoleOthers)
-		table.AddRecord(record2)
+func (n *layer) GetContainerEACL(ctx context.Context, cid *cid.ID) (*eacl.Table, error) {
+	signedEacl, err := n.pool.GetEACL(ctx, cid)
```

Comment:
Can you create an issue on caching of EACL values? I am not sure if we need that, because it is already cached in Storage node, thus we can discuss it there.

Reply:
#202